charts/operator-wandb/values.yaml

```diff
@@ -101,15 +101,13 @@ app:
 
 weave:
   install: true
-  # # azurefile, filestore
-  # provider: ""
-  storageClass: ""
   image:
     repository: wandb/local
     tag: latest
-  persistence:
-    efs:
-      fileSystemId: ""
+  cache:
+    intervalInHours: 24
+    size: 20Gi
+    medium: ""
```

Comment:
Sorry for all the nit stuff, we should leave it in the weave config and not in the root values.yaml. Its inherired by default this way we can chagne it in once place